src/main/java/org/desarrolladorslp/technovation/controller/EventController.java

```diff
@@ -0,0 +1,30 @@
+package org.desarrolladorslp.technovation.controller;
+
+import org.desarrolladorslp.technovation.controller.dto.EventDTO;
+import org.desarrolladorslp.technovation.services.EventService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("api/events/{year}/{month}")
```

Comment:
The RequestMapping here is only `api/events` the other part is specific for the method